google-cloud-tools-plugin/build.gradle

```diff
@@ -49,6 +49,7 @@ dependencies {
 
     compile 'com.google.apis:google-api-services-clouddebugger:v2-rev7-1.21.0'
     compile 'com.google.apis:google-api-services-cloudresourcemanager:v1beta1-rev12-1.21.0'
+    compile 'com.google.apis:google-api-services-appengine:v1-rev8-1.22.0'
```

Comment:
actually I take that back. I guess it makes sense and CRM is just for non product specific entities.